official/vision/beta/projects/deepmac_maskrcnn/serving/export_base.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2021 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Why is this file required ? We could just directly import `export_base` in `detection.py`

Reply:
Deleted the file. Also removed the import from detection.py as it's not required.